### PR TITLE
Bump zwave-js to 9.0.2 and zwave-js-server to 1.16.0

### PR DESCRIPTION
**Pull request by raman325**
~~This PR isn't working correctly yet which is why it's still in draft. Details after changelog.~~ This has been resolved. According to this issue we need to build @serialport/bindings-cpp from source: https://github.com/serialport/node-serialport/issues/2438

Changelog:
- https://github.com/zwave-js/node-zwave-js/releases/tag/v9.0.2
- https://github.com/zwave-js/node-zwave-js/releases/tag/v9.0.1
- https://github.com/zwave-js/node-zwave-js/releases/tag/v9.0.0

### CODE REVIEW

**Review by MartinHjelmare — APPROVED** [review]
I wonder if we should delay releasing this a couple of days after the core release to not potentially confuse users about what could be the reason for any issues that come up with their devices?

**Comment by frenck** [comment]
🤷 I'm happy to adopt any schedule

**Comment by raman325** [comment]
It's been a couple of days and no issues. I think we can merge?

**Comment by raman325** [comment]
this is ready to be merged.

File: zwave_js/config.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
----
```
Comment (MartinHjelmare) [inline zwave_js/config.yaml:1]: This should stay I think.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Comment by raman325** [comment]
can someone restart CI?

**Comment by MartinHjelmare** [comment]
Another job failed this time.

**Comment by raman325** [comment]
`  Error: fatal: Could not read from remote repository.` Seems like maybe a problem with CI and not with the PR?

**Comment by raman325** [comment]
all good now!

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**